configure.ac

```diff
@@ -1,4 +1,4 @@
-AC_INIT([libaparapi], [1.2.1], [syncleus@syncleus.com])
+AC_INIT([libaparapi], [1.2.2], [syncleus@syncleus.com])
```

Comment:
this should be 1.3.2 but technically it is ignored when generating the library, the variables that determine the version are elsewhere (though i already bumped them). Might as well make this 1.3.2 though